internal/cmd/test/test.go

```diff
@@ -156,11 +156,12 @@ func (m mFlagRunner) Run() error {
 			continue
 		}
 
+		cmd := newTestCmd(m.args)
```

Comment:
a command should be built within the app's root so the correct configuration was applied.